nva/nva.c

```diff
@@ -135,6 +147,84 @@ struct nva_card *nva_init_apu(struct pci_device *dev) {
 	return card;
 }
 
+static int check_modalias(const char *node_name) {
+	char path[PATH_MAX];
+	char buffer[255] = { 0 };
+	FILE *file;
+
+	/* Check if the GPU is indeed made by NVIDIA by checking the compatibility
```

Comment:
Yes, I purposely wanted to support pre-K1 tegra GPUs. Is it a problem? Sure, they are different but someone may want to take a peek at them at some point, right?